ArmTemplates/APIM/apim-api.json

```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "apimProductInstanceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The product identifier, this is different to the Display Name (which may contain spaces)."
```

Comment:
Where is the display name set? This might be one for the documentation but I didn't see a apim-api.md file

Reply:
Set in different template, added documentation